### DISCUSSION TITLE
report artillery results to Allure

### DISCUSSION THREAD

**Discussion by iasisalomon** [artilleryio/artillery#1209] [Artillery]
We are using allure for test reporting on multiple areas (pentest, e2e, etc.) and we would love to add stress testing results to allure reports.

We are using https://github.com/fescobar/allure-docker-service#info-endpoints for this matter.

What is the most convininent way, if any, to integrate both?

**Comment by hassy** [comment]
That sounds like something that could be done with a plugin, similar to how [`publish-metrics`](https://github.com/artilleryio/artillery-plugin-publish-metrics) sends results to monitoring systems. We have some documentation on plugin development here: https://artillery.io/docs/guides/guides/extending.html#Plugin-events - would be happy to help out / review code!

**Reply by Balramidigitalxp** [reply]
https://artillery.io/docs/guides/guides/extending.html#Plugin-events This link is broken

**Reply by hassy** [reply]
updated link: https://www.artillery.io/docs/guides/guides/extension-apis
